lib/bank_accounts/account.rb

```diff
@@ -0,0 +1,45 @@
+require 'pry'
+
+module BankAccounts
+  class Account
+
+    attr_reader :id, :balance
+
+    # Rounds float value down to hundredths position
+    # For example 10.2599 rounds to 10.25
+    def round_to_hundredths(n)
+      n = (((n * 100).floor) / 100.0)
+      return n
```

Comment:
Hi @roseaboveit! I was adding that just to be explicit. Is this something that other devs/programmers dislike?